### PR TITLE
fix: consent to all cookies before logging in

### PR DESCRIPTION
**Pull request by michelalbers**
## Description

Instagram displays a cookie consent message before we can login. This fix clicks the "accept all cookies" button before moving on.

Fixes # (issue)

#6234 

## How Has This Been Tested?

Its now possible to login

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project, `black -t py34`
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by fintzd — COMMENTED** [review]
Hey! Thanks for that, there has been many comments about this error in the issues section, however, it seemed to me as if this should have been already fixed by a previous PR, could you check if you have the latest version of instapy installed and that it does not work? 

The last PR that has been merged and which seems to be handling this issue is [#6079](https://github.com/timgrossmann/InstaPy/pull/6079). Did you check that out?

**Comment by ntelo007** [comment]
I just made the change locally and I can verify that it works only with this merge.

**Comment by hadifarnoud** [comment]
why isn't this merged yet?

File: instapy/login_util.py [hunk 1]
```diff
@@ -242,6 +242,15 @@ def login_user(
     ig_homepage = "https://www.instagram.com"
     web_address_navigator(browser, ig_homepage)
 
+    for element in browser.find_elements_by_tag_name('button'):
+        if element.text.strip().lower() == 'accept all':
```
Comment (hadifarnoud) [inline instapy/login_util.py:246]: it does not work with new changes

**Review by hadifarnoud — CHANGES_REQUESTED** [review]

**Pull request closed without merging**